lexicons/com/atproto/admin/defs.json

```diff
@@ -368,6 +380,145 @@
         "height": { "type": "integer" },
         "length": { "type": "integer" }
       }
+    },
+    "subjectReviewState": {
+      "type": "string",
+      "knownValues": ["#reviewOpen", "#reviewEscalated", "#reviewClosed"]
+    },
+    "reviewOpen": {
+      "type": "token",
+      "description": "Moderator review status of a subject: Open. Indicates that the subject needs to be reviewed by a moderator"
+    },
+    "reviewEscalated": {
+      "type": "token",
+      "description": "Moderator review status of a subject: Escalated. Indicates that the subject was escalated for review by a moderator"
+    },
+    "reviewClosed": {
+      "type": "token",
+      "description": "Moderator review status of a subject: Closed. Indicates that the subject was already reviewed and resolved by a moderator"
+    },
+    "modEventTakedown": {
+      "type": "object",
+      "description": "Take down a subject permanently or temporarily",
+      "properties": {
+        "durationInHours": {
+          "type": "integer",
+          "description": "Indicates how long the takedown should be in effect before automatically expiring."
+        },
+        "expiresAt": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Indicates at what time the subject's takedown will be/was reverted."
+        }
+      }
+    },
+    "modEventReverseTakedown": {
```

Comment:
It kind of feels like there could be just a single `modEventTakedown` with a boolean flag. reversing would be emitting that even with takedown=false. Same with mute. This would align with how we negate labels.

I could go either way on it though, it's ok how it is now as well.